client/src/lib/player.js

```diff
@@ -1,61 +1,70 @@
-import * as Tone from 'tone'
+import { duration } from '@mui/material';
+import * as Tone from 'tone';
 
 export class Layer {
   constructor({ url, volume, pitch, id, layerData }) {
     this.id = id;
     this.url = url;
-    this.player = new Tone.Player(this.url)
+    this.player = new Tone.Player(this.url);
     this.volume = new Tone.Volume(volume || -5);
-    this.pitchShift = new Tone.PitchShift(pitch || 0)
+    this.pitchShift = new Tone.PitchShift(pitch || 0);
     this.waveform = new Tone.Waveform();
     this.solo = new Tone.Solo().toDestination();
-    this.layerData = layerData
-    this.name = getLayerName(this.layerData)
-
-    this._pitch = this.pitchShift.pitch
-    this._mute = false
-    this._solo = false
-    this._volume = this.volume.volume.value
+    this.layerData = layerData;
+    this.name = getLayerName(this.layerData);
+
+    this.trimFromStart = this.layerData.trimFromStart || 0;
```

Comment:
Minor but we can probably add 'trimFromStart' to the params in the constructor instead of getting it from layerData since layerData's object is being spread as the input params.